src/ol_infrastructure/infrastructure/aws/eks/__main__.py

```diff
@@ -53,6 +53,7 @@
     "EBS_CSI_DRIVER": "v1.33.0-eksbuild.1",
     "EFS_CSI_DRIVER": "v2.0.7-eksbuild.1",
     "VAULT_SECRETS_OPERATOR": "0.8.1",
+    "OPEN_METADATA": "1.5.5",
```

Comment:
This doesn't seem like it should go here?

Reply:
This should probably go [here](https://github.com/mitodl/ol-infrastructure/blob/main/src/bridge/lib/versions.py) (maybe this entire dict should be moved into `versions.py`?) and eventually we should make it/them envrionment var loads so that the stacks can be updated to latest versions automagically by the pipeline (eventually).

For now, probably just move this to `versions.py` so that we aren't creating an unusual relationship between this application stack and the infrastructure stack.

Reply:
Looks like mike moved this to VERSIONS. Thanks Mike!